features/dd-sdk-android-trace/transitiveDependencies

```diff
@@ -1,13 +1,28 @@
 Dependencies List
 
 androidx.annotation:annotation:1.3.0                            :   30 Kb
+com.github.spotbugs:spotbugs-annotations:4.2.0                  :   14 Kb
+com.github.spotbugs:spotbugs:4.2.0                              :    3 Mb
```

Comment:
Is it a runtime or compile time dependency? If it exists in the final artifact, we need to remove it.

Reply:
it is compile time